### PR TITLE
Update README to match project

### LINKED ISSUE #25: Update README to match project

### PR DESCRIPTION
**Pull request by kvihashini**
Fix #25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,17 +1,23 @@
...
-* This project is a **part of the se-education.org** initiative. If you would like to contribute code to this project, see [se-education.org](https://se-education.org#https://se-education.org/#contributing) for more info.
+TutorMaster is a desktop app for freelance tutors to manage their students’ grades, assignment submission and tuition fee payment. It is optimised for use via a Command Line Interface (CLI) while still benefiting from a Graphical User Interface (GUI).
+
+## Features
+- Handle Students
+  - Add
+  - Edit
+  - View
+  - Delete
+  - Group
```
Comment (Neha-5678) [inline README.md:15]: Just a minor detail, I think you may have missed out the list all students and find student feature!

File: README.md [hunk 2]
```diff
@@ -1,17 +1,23 @@
...
+- Handle Tasks
+  - Add
+  - Edit
+  - Delete
+  - Assign to Student
```
Comment (Neha-5678) [inline README.md:20]: Can consider adding the View task feature as well :)

**Review by Neha-5678 — COMMENTED** [review]
LGTM! Just some nits to fix.

File: README.md (hunk 1 above)
Comment (kvihashini) [reply README.md]: I have added it! Thanks for your suggestion 👍🏽

File: README.md (hunk 2 above)
Comment (kvihashini) [reply README.md]: I have added it too! Thank you 👍🏽

**Review by Neha-5678 — APPROVED** [review]
LGTM!

**Pull request merged**